app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java

```diff
@@ -1,23 +1,94 @@
 package ar.com.wolox.android.example.ui.example.login;
 
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
 import ar.com.wolox.android.R;
+import ar.com.wolox.android.example.ui.example.ExampleActivity;
 import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+import butterknife.BindView;
 
 /**
  * A simple {@link WolmoFragment} subclass.
  */
 public class LoginFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
 
+    @BindView(R.id.vLoginEmailEditText)
+    EditText vLoginEmailEditText;
+
+    @BindView(R.id.vLoginPasswordEditText)
+    EditText vLoginPasswordEditText;
+
+    @BindView(R.id.vLoginLogInButton)
+    Button vLoginLogInButton;
+
     @Override
-    public void init() {}
+    public void init() {
+        loadLoginData();
+    }
+
+    private void loadLoginData() {
+        Context context = getActivity();
```

Comment:
There is no need to use this line. You can use directly getContext() method

Reply:
✔ **Done**
Interesting correction